src/components/navigation-user/navigation-user.tsx

```diff
@@ -0,0 +1,81 @@
+import { Component, Element, h, Host, Prop, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-navigation-user",
+  styleUrl: "navigation-user.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavigationUser {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteNavigationUserElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted.*/
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the text to display, such as a username or full name.*/
+  @Prop() fullName: string;
+
+  /** Describes the appearance of the avatar. If no label is provided, context will not be provided to assistive technologies.*/
+  @Prop() label: string;
+
+  /** When `true`, hides the `fullName` and `username` contents.*/
+  @Prop({ reflect: true }) textDisabled = false;
+
+  /** Specifies the `src` to an image to display in the Avatar.*/
+  @Prop() thumbnail: string;
+
+  /** Specifies the unique id of the user.*/
+  @Prop() userId: string;
+
+  /** Specifies the username of the user.*/
+  @Prop() username: string;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  render(): VNode {
+    return (
+      <Host>
+        <button aria-label={this.label}>
+          <calcite-avatar
```

Comment:
note: I wonder if it might make more sense to just have them slot the `<calcite-avatar>` instead of this component having all the same props?

Reply:
The reason for having a separate component is to restrict the users from setting the scale which can cause layout issues in navigation context me thinks.  We can considering re-using `avatar` component by rendering `username` & `fullName`  in the DOM and adjusting scaling based on CSS selectors to identify if its slotted inside navigation component.

WDYT @macandcheese @jcfranco

Reply:
We could always modify the scale of the avatar onslotchange.